test/infrastructure/docker/config/infrastructure.cluster.x-k8s.io_dockerclusters.yaml

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
Shouldn't these CRDs be under `test/infrastructure/docker/config/crd/bases` ?

Reply:
You're right, i had messed up the controller-gen command. It's fixed and generating in the correct spot now.